swarmit/src/io/polyswarm/swarmit/optionspanel/SwarmItMarketplaceSettings.java

```diff
@@ -36,18 +36,19 @@
  */
 public final class SwarmItMarketplaceSettings {
     
+    
     private final static Logger LOGGER = Logger.getLogger(SwarmItMarketplaceSettings.class.getName());
-    private final Double DEFAULT_NCT_AMOUNT = 0.5;
-    private final String DEFAULT_API_KEY = "PLACEHOLDER0123456789ABCDEF"; // NON-NLS
-    private final String DEFAULT_URL = "https://consumer.prod.polyswarm.network/"; // NON-NLS
+    private final String DEFAULT_API_KEY = ""; // NON-NLS
+    private final String DEFAULT_URL = "https://api.polyswarm.network/v2"; // NON-NLS
+    private final String DEFAULT_COMMUNITY = "omicron"; // NON-NLS

```

Comment:
^ set to "default" ?

Reply:
Yea, I added this because I wasn't able to get that working on stage. It's probably working now. Will test.